embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/injection/OpenTelemetryModule.kt

```diff
@@ -13,11 +13,19 @@ import io.embrace.android.embracesdk.internal.spans.SpansSink
 import io.embrace.android.embracesdk.internal.spans.SpansSinkImpl
 import io.embrace.android.embracesdk.opentelemetry.OpenTelemetrySdk
 import io.opentelemetry.api.trace.Tracer
+import io.opentelemetry.sdk.trace.export.SpanExporter
 
 /**
  * Module that instantiates various OpenTelemetry related components
  */
 internal interface OpenTelemetryModule {
+    /**
+     * Adds one [SpanExporter] to the tracer.
+     *
+     * @param spanExporter the span exporter to add
+     */
+    fun addSpanExporter(spanExporter: SpanExporter)
```

Comment:
The module shouldn't carry state - it's just there to initialize services and hang objects private to those services off of. I would probably create a new object/service that we can write all of the custom stuff we need to initialize the OTel SDK with, like the SpanExporter, the LogExporter eventually, and additional attributes.

For now, just add support for SpanExporter and we can add to it when we add new features?